app/models/talk.rb

```diff
@@ -41,26 +52,56 @@ class Talk < ApplicationRecord
   # delegates
   delegate :name, to: :event, prefix: true, allow_nil: true
 
+  before_save :compute_embedding, if: :must_compute_embedding?
+
   # search
-  meilisearch do
+  meilisearch primary_key: :id, enqueue: true, raise_on_failure: Rails.env.development? do
     attribute :title
     attribute :description
     attribute :slug
-    attribute :video_id
+    # ⚠️ This `video_id` attribute makes indexing (silently) fail with v1.3.2 of meilisearch. Error message from meilisearch (GET /tasks):
```

Comment:
@Kerollmops I wonder if there is a regression here with v1.3.2, since it fails to index even if I specifically set the `primary_key: :id` as above.

Reply:
I recently specified the primary key to index one of my datasets and haven't had issues. Are you sure it was introduced in v1.3.2? Is it related to the ruby SDK? I used the `primaryKey` query parameter with the `POST /indexes/{indexname}/documents` route.

Reply:
hi @crohr why do you need to specify the primary key?

Reply:
It was just an attempt to fix the issue I'm explaining in the comment, but it doesn't make a difference (and shouldn't since this is the default)